internal/pkg/utils/testhelper/terminal/terminal_linux.go

```diff
@@ -138,7 +138,7 @@ func (c *console) waitBeforeSend() {
 	time.Sleep(sendDelay)
 }
 
-func (t *tty) Read(p []byte) (n int, err error) {
+func (t *tty) Read(p []byte) (int, error) {
```

Comment:
Fix race condition, ...

Ak su parametre definovane hore ^^^, tak pri return sa do nich zapisuje.
Rovnako sa do nich zapisuje aj v goroutine.

Race detector -u sa to nepacilo, ze to nie je chranene zamkom, aj ked realne to nie je problem.